target_chains/solana/programs/pyth-solana-receiver/src/lib.rs

```diff
@@ -340,6 +383,24 @@ pub struct ReclaimRent<'info> {
     pub price_update_account: Account<'info, PriceUpdateV2>,
 }
 
+// New context for initializing PriceUpdateV2 without verifications
+#[derive(Accounts)]
+pub struct InitializePriceUpdateV2<'info> {
+    #[account(init, payer = user, space = PriceUpdateV2::LEN)]
```

Comment:
You can use `init_if_needed` and you will only need 1 instruction

Reply:
Thanks